src/app/clients/add/submitFormHelpers.ts

```diff
@@ -116,10 +108,7 @@ const deleteChildren = async (children: Person[]): Promise<void> => {
             .eq("primary_key", child.primaryKey);
 
         if (errorExists(error, status)) {
-            // TODO VFB-22 Standardize Error Handling
-            throw Error(
-                `Error occurred whilst deleting children in the Families table. HTTP Code: ${status}, PostgreSQL Code: ${error?.code}. `
-            );
+            throw new Error(RequestErrorMessage);
```

Comment:
For these probably just make a `RequestError` class that extends error so you can just throw that. It also means you can define a more meaningful constructor and unify formatting for error messages, and users elsewhere can specifically check for and catch that error.